lib/markdown.dart

```diff
@@ -68,6 +68,7 @@ export 'src/inline_syntaxes/autolink_extension_syntax.dart';
 export 'src/inline_syntaxes/autolink_syntax.dart';
 export 'src/inline_syntaxes/code_syntax.dart';
 export 'src/inline_syntaxes/color_swatch_syntax.dart';
+export 'src/inline_syntaxes/decode_html_syntax.dart';
```

Comment:
Do we WANT this exported? Should all syntaxes be exported? This was in the changelog, but not exported...

Reply:
I suggest to export it. I forgot it.